.github/workflows/Add Pool.yml

```diff
@@ -0,0 +1,72 @@
+name: Add new Pool to xSPO-Alliance
+
+on:
+  issues:
+    types: [opened, reopened]
+
+jobs:
+  issue-to-json:
+    name: Convert issue body to json
+    if: contains(github.event.issue.body, '>>pool_bech32_id<<')
+    runs-on: ubuntu-latest
+    outputs:
+      payload-parsed: ${{ steps.payload.outputs.json_var }}
+    steps:
+      - name: Run parser
+        id: parse
+        uses: peter-murray/issue-forms-body-parser@v2.0.0
+        with:
+          issue_id: ${{ github.event.issue.number }}
+          separator: '###'
+          label_marker_start: '>>'
+          label_marker_end: '<<'
+      - name: Set Output
+        id: payload
+        run: echo json_var='${{ steps.parse.outputs.payload }}' >> $GITHUB_OUTPUT
+
+  execute-pipe:
+    name: Adding new Pool
+    if: contains(github.event.issue.body, '>>pool_bech32_id<<')
+    runs-on: ubuntu-latest
+    needs: issue-to-json
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        #### Important, in case main branch is protected (for direct commits) it has to checkout in a valid branch
+        ### In this case, change below
+        #with:
+        #  ref: '<branch_name>'
+     
+      - name: 'Pool add'
+        id: status
+        continue-on-error: true
+        env:
+          pool_bech32_id: ${{ fromJson(needs.issue-to-json.outputs.payload-parsed)['pool_bech32_id'] }}
+        run: ./scripts/add-pool.sh          
+
+      - name: "commit"
+        uses: EndBug/add-and-commit@v9
+        if: steps.status.outcome == 'success'
```

Comment:
commit: only commits the changes if previous step is succeed
Critical step because wrong commits will break everything